made-with.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default.html
+title: Made With Servo
+---
+
+<style type="text/css">
+
+.made-with-container {
+  gap: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  margin-bottom: 32px;
+}
+
+.made-with-item {
+  display: block;
+  width: 400px;
+  flex: 0 0 400px;
+  border: 2px solid #333;
+  cursor: pointer;
+  color: inherit;
+  text-decoration: inherit;
+}
+
+.made-with-item:hover {
+  background-color: rgba(27, 131, 129, 0.5);
+  background-color: #EAEAEA;
+}
+
+.made-with-text {
+  padding: 20px;
+}
+
+.made-with-text > p {
+  padding: 0;
+}
+
+.made-with-image {
+  aspect-ratio: 2;
+  object-fit: cover;
+  object-position: top left;
+  background-color: transparent;
+}
+
+</style>
+
+<div class="inner-container">
```

Comment:
Additionally add class "subpage-content" and this will resolve spacing issues on the rest of the page and inline styles can be removed.

Reply:
"subpage-content" restricts the width which makes sense for textual content but doesn't really make sense for these "boxes". So I've created a new "fullwidth" variant which doesn't do that but applies the rest of the "subpage-content" styles.